templates/display_product.tt.html

```diff
@@ -16,6 +16,12 @@
     </a>
 </div>
 
+<!-- Show graph page -->
+<div class="edit_button right" style="float:right;margin-top:-10px;margin-left: 10px;">
+  <a href="/cgi/graph.pl?barcode=[% code %]&country=&score=nutriscore" class="button small">
+	<span class="show-for-large-up">Graph</span>
+</a></div>
```

Comment:
Do we need some kind of server option to hide/show this? The graphs feature may not be interesting on open products facts or open beauty facts.

Reply:
The buttons on top of product pages are meant for actions that change the product page (edit or delete), so I would prefer to show the link to the graph somewhere else.

I think it would be good to find a way to tell users that this is an experimental feature (a kind of "lab"), and that it may not be as documented etc. as the more classic OFF features. Kind of like the browser extensions / user scripts that others have done and that add features to OFF when the extension is active.

Instead of "graph", we should probably use something more descriptive, e.g. "Explore similar products" or even "See the Nutri-Score and NOVA group of similar products".

Reply:
Maybe a good place could be where we display the Nutri-Score:

![image](https://user-images.githubusercontent.com/8158668/92327319-0b444300-f059-11ea-97c4-368fa316d61a.png)